e2e/cluster/local/docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: "3.7"
+
+volumes:
+  kubeconfig:
+    name: e2e-local-kubeconfig
+
+services:
+
+  leader:
+    container_name: local-leader
+    image: "e2e/cluster/local/k3s"
+    build:
+      context: ./images/k3s/.
+      args:
+        - ARCH=${ARCH}
+    command: server --no-deploy=traefik,metrics-server
```

Comment:
Can we use disable here instead of no-deploy

Reply:
`--disable` doesn't yet seem to be available in a release

Reply:
got it